model/cli/spans.yaml

```diff
@@ -0,0 +1,14 @@
+groups:
+  - id: span.cli
+    type: attribute_group
+    stability: experimental
+    brief: >
+      Describes CLI (Command Line Interfaces) programs span.
+    attributes:
+      - ref: process.command_line
```

Comment:
what about executable name? pid? owner?

Reply:
How many attributes do we need to include before CLI semconv becomes equivalent to process semconv? I was under the assumption we would only extract the "command" related attributes.

Reply:
process semconv describes resource attributes - not spans. Here I believe we're trying to specify which attributes would be populated on spans, what's the span name and kind is.

And why would someone not want process pid on the corresponding span? 

Also, maybe the answer is that process attributes are set as resource attributes on that span and then here we'd just add a link to process resource attributes saying that they should be set?

Reply:
Updated the list of attributes